pkg/auth/providerrefresh/refresh_daemon.go

```diff
@@ -74,6 +78,10 @@ func UpdateRefreshCronTime(refreshCronTime string) {
 }
 
 func UpdateRefreshMaxAge(maxAge string) {
+	if ref == nil {
+		return
+	}
+
 	ref.maxAge = parseMaxAge(maxAge)
```

Comment:
This should have a validator added and if the input is bunk use the previous value. Changing to 0 will have consequences the user probably won't like.